### ISSUE TITLE
fix: [Narrator Does not announce the min and Max values] in [Slider]

### ISSUE THREAD

**Issue by shachak** [microsoft/fast#5671]
# 🐛 Bug Report

Narrator Should announce the min max value of the slider along with the current value.

## 💻 Repro or Code Sample

Can be repro-ed on component explorer

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* Version [e.g. 1.8.0]

**Comment by EisenbergEffect** [comment]
@shachak The labels we add need to be meaningful for the global community. Thus, we've labeled this with our all7 compliance label. /cc @chrisdholt  for resolution.

**Comment by chrisdholt** [comment]
@shachak Do you repro what is expected with the native input here? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

On my Mac using VoiceOver I repro this same lack of behavior for both our slider and a native input. I'm not sure this is something that's able to be fixed from our implementation. After investigating, the expectation is that both `aria-valuemin` and `aria-valuemax` are both present on the element with a role of slider. The implication is that AT will ensure that these are read out. What I'm seeing with native inputs as well here makes me think that this is not an issue specific to our implementation.

**Comment by shachak** [comment]
> @shachak Do you repro what is expected with the native input here? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range
> 
> On my Mac using VoiceOver I repro this same lack of behavior for both our slider and a native input. I'm not sure this is something that's able to be fixed from our implementation. After investigating, the expectation is that both `aria-valuemin` and `aria-valuemax` are both present on the element with a role of slider. The implication is that AT will ensure that these are read out. What I'm seeing with native inputs as well here makes me think that this is not an issue specific to our implementation.

@chrisdholt I am seeing the same issue with the native input here as well. So, I am not sure what you mean, but is it the issue with the underlying native input? Given, that you say we are adding both aria-valuemin and aria-valuemax.

**Comment by chrisdholt** [comment]
> > @shachak Do you repro what is expected with the native input here? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range
> > On my Mac using VoiceOver I repro this same lack of behavior for both our slider and a native input. I'm not sure this is something that's able to be fixed from our implementation. After investigating, the expectation is that both `aria-valuemin` and `aria-valuemax` are both present on the element with a role of slider. The implication is that AT will ensure that these are read out. What I'm seeing with native inputs as well here makes me think that this is not an issue specific to our implementation.
> 
> @chrisdholt I am seeing the same issue with the native input here as well. So, I am not sure what you mean, but is it the issue with the underlying native input? Given, that you say we are adding both aria-valuemin and aria-valuemax.

The reason I bring this up is because I don't think it's a bug that's on our implementation. Our implementation is not using the native input but the role of slider and ARIA. The fact that this repro's in the native element is telling, as if this is expected behavior here, the issue repro's in both implementations. We've built to meet the requirements outlined, and the native input is written to meet the specification as well - so I believe that this is likely a bug on Narrator (and Voiceover from my repro).

**Comment by chrisdholt** [comment]
Closing this given the above response. If you believe this has been closed in error, please feel free to reopen and reach out.
